framework/src/utils/RankThreeTensor.C

```diff
@@ -252,6 +264,38 @@ RankThreeTensor::fillFromInputVector(const std::vector<Real> & input, FillMethod
 }
 
 void
+RankThreeTensor::fillFromPlaneNormal(const TypeVector<Real> & input)
```

Comment:
In moose `TypeVector<Real>` is used 15 times, whereas `RealVectorValue` is used 838 times. I'd suggest to drop the odd one out.